### DISCUSSION TITLE
Continuous sythesis improvment

### DISCUSSION THREAD

**Discussion by twsowerby** [coqui-ai/TTS#1669] [General Q&A]
Hey all,

I'm interested to learn what your process is for continually improving a model? 

I've managed to pretty successfully train tacotron2 and VITS to replicate a voice, however I notice the quality of synthesis is a lot better when you synthesise one of the text lines it was trained on (rather than a made up line of text that isnt in the training dataset). 

This makes sense, however I'm keen to understand how you all go about continuing to fine tune the model to expand its capability

Do I use the most recent finetuned model and add more to the dataset and train again? Do I feed it a completely new dataset (but be careful to maintain consistent speaking style)? Do I supplement the original dataset with what I "missed" (words, pronunciation, style, prosody), and start from scratch?

Thanks!

**Comment by p0p4k** [comment]
You might refer to this #1230

> IF you only finetune with the new dataset, it might start to forget the previous set of voices it has learned. If your intention is to increase the number of speakers, then it is better to merge the dataset and train the model with all the samples.
> 
> If you only want to use it with the new speaker, then you can keep training with only the new dataset.

Please use search function in discussions to avoid repeating older/similar questions. Thanks. Good luck! 👍
